test/mocha/number-literal.js

```diff
@@ -21,4 +21,46 @@ describe("Number literals", function () {
             assert.throws(test(inputs[i]), error, inputs[i]);
         }
     });
+    it("should correctly parse BigInt", () => {
+        const inputs = [
+            "console.log(100000n);",
+            "console.log(-15n);",
+            "console.log(0x20000000000001n);",
+            "console.log(0b101n);",
+            "console.log(0o7n);"
+        ];
+        const outputs = [
+            "console.log(100000n);",
+            "console.log(-15n);",
+            "console.log(0x20000000000001n);",
+            "console.log(0b101n);",
+            "console.log(0o7n);"
```

Comment:
Could this test be rewritten as a compress test with an expect_exact clause? This feature warrants the creation of test/compress/bigint.js

Reply:
I tried but it does not work:

*test/compress/bigint.js*:
```javascript
bigint_positive: {
    input: {
        console.log(100000n);
    }
    expect_exact: "console.log(100000n);"
}
```

*error*:
```
--- bigint.js
    Running test [bigint_positive]
!!! Mozilla AST I/O corrupted input
---INPUT---
console.log(100000n);
---OUTPUT---
console.log("100000");
```

Reply:
I think this is because the Mozilla AST conversion for bigint is not done yet. I think it might be mixing your number up with a string.

Reply:
Fixed :)